code/src/index.css

```diff
@@ -1,13 +1,221 @@
-body {
+/* Mobile version */
+* {
+  padding: 0;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  box-sizing: border-box;
+}
+
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+  text-decoration-color: white;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.horizontalLine {
+  display: block;
+  color: #e9f4fe;
+  border-style: inset;
+  opacity: 0.2;
+}
+.main-container {
+  padding: 1em;
+}
+
+/* ------MainHeader------*/
+.mainHeading {
+  font-size: 25px;
+  color: white;
+  padding: 1em 0 1em;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.headerLogo {
+  width: 100px;
+}
+
+/* ----- ICONS -----*/
+
+.icons {
+  width: 40px;
+  z-index: 10;
+  position: absolute;
+  opacity: 0.5;
+  cursor: pointer;
+}
+
+/* ----- FOOTER -----*/
+
+.madeBy {
+  font-size: 15px;
+  text-align: center;
+  color: white;
+  padding: 1em;
+}
+.Names {
+  color: white;
+}
+.links {
+  text-decoration: none;
+}
+
+/* ----- album -----*/
+.album-container {
+  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  margin: 1em 0 1em 0;
+  background-color: black;
+}
+.album-card {
+  width: 300px;
+  height: 400px;
+  position: relative;
+  margin: 1em;
+}
+.music-container {
+  padding: 1em;
+  justify-items: center;
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.album-image-container {
+  position: relative;
+  display: flex;
+}
+
+.album-cover {
+  opacity: 1;
+  transition: 0.5s ease-in;
+  z-index: 1;
+}
+
+.overlay:hover {
+  opacity: 0.3;
+}
+
+.album-title {
+  padding-top: 18px;
+  padding-bottom: 8px;
+  font-size: 18px;
+}
+
+.album-title-link {
+  color: white;
+  text-decoration: none;
+}
+
+/* ----- singer -----*/
+.singer {
+  display: inline-block;
+  color: white;
+  font-size: 14px;
+  text-decoration: none;
+}
+
+.singer:not(:first-of-type, :last-of-type)::before {
+  content: ', ';
+}
+
+.singer:nth-last-of-type(2)::after {
+  content: ' & ';
+}
+
+.singer:first-child::before {
+  content: ' ';
+}
+
+/* ----- icons -----*/
+
+.icons {
+  display: flex;
+  position: absolute;
+  padding: 5px;
+  z-index: 12;
+  position: absolute;
+  opacity: 0.5;
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  top: 40%;
+  left: 45%;
+  opacity: 0;
+  grid-gap: 30px;
+  gap: 30px;
+}
```

Comment:
Here you repeat the .icons. Is there a specific reason why? Maybe merge rhis part with the .icons part higher up so you don't repeat youself